lib/routes.js

```diff
@@ -0,0 +1,15 @@
+var routes = {
+  method: "GET",
+  path:   "/version",
+  config: {
+    state: {
+      parse:      true,
+      failAction: "log"
```

Comment:
this whole `config` thing was a workaround to an issue i had, without it I was getting `{"statusCode":400,"error":"Bad Request","message":"Invalid cookie value"}` from the server, with it, I don't.

On further inspection, maybe there's another issue here? If I change this to `"error"` I get that error above, if I leave it at `"log"`, I get the `{"version":"0.0.1"}` response I'd expect.